app/controllers/appointment_controller.js

```diff
@@ -0,0 +1,612 @@
+                id: faker.random.uuid(),
+                routeDetails: {
+                    controller: 'Appointment',
+                    action: 'index_time_slot'
+                },
+                requestedTimeSlot: [
+                    {
+                        validFor: {
+                            startDateTime: new Date(),
+                            endDateTime: new Date()    
+                        }
+                    }
+                ],
+                relatedParty: {
+                    id: faker.random.uuid(),
+                    href: faker.internet.url(),
+                    name: "string",
+                    role: "string",
+                },
+                place: {
+                    role: "interventionAddress",
+                    "@referredType": "geographicAddress",
+                    geographicAddress: {
+                        streetNr: faker.random.number(),
+                        streetType: "Avenue",
+                        streetName: "string",
+                        postCode: faker.random.number(),
+                        city: "Paris",
+                        country: "France" 
+                    }
+                },
+                relatedEntity: [
+                    {
+                        id: faker.random.uuid(),
+                        href: faker.internet.url(),
+                        name: "Internet connexion problem to solve",
+                        "@referredType": "troubleTicket"
+                    }
+                ]
+            }
+        ]
+    },
+    show_time_slot: function(request)
+    {
+        console.log("appointment_controller.show_time_slot")
+        return {
+            id: request.params.slot_id,
+            routeDetails: {
+                controller: 'Appointment',
+                action: 'show_time_slot'
+            },
+            requestedTimeSlot: [
+                {
+                    validFor: {
+                        startDateTime: new Date(),
+                        endDateTime: new Date()    
+                    }
+                }
+            ],
+            relatedParty: {
+                id: faker.random.uuid(),
+                href: faker.internet.url(),
+                name: "string",
+                role: "string",
+            },
+            place: {
+                role: "interventionAddress",
+                "@referredType": "geographicAddress",
+                geographicAddress: {
+                    streetNr: faker.random.number(),
+                    streetType: "Avenue",
+                    streetName: "string",
+                    postCode: faker.random.number(),
+                    city: "Paris",
+                    country: "France" 
+                }
+            },
+            relatedEntity: [
+                {
+                    id: faker.random.uuid(),
+                    href: faker.internet.url(),
+                    name: "Internet connexion problem to solve",
```

Comment:
```suggestion
                    name: "Internet connection problem to solve",
```

There are a few other places that this comes up, please check for them. Sorry this is nitpicky

Reply:
Fixed